libmscore/measure.cpp

```diff
@@ -2507,6 +2507,20 @@ bool Measure::isFirstInSystem() const
     return system()->firstMeasure() == this;
 }
 
+//---------------------------------------------------------
+//   scanElements
+//---------------------------------------------------------
+
+void Measure::scanElements(void* data, void (* func)(void*, Element*), bool all)
+{
+    for (ScoreElement* el : (*this)) {
+        if (el == nullptr || el->type() == ElementType::MEASURE) {
```

Comment:
What about this?
```c++
if (el == nullptr || el->isMeasure()) {

```

Also, can you name a situation where a Measure is child of a Measure?

Reply:
For the sake of those following, a multi-measure rest (MMRest) is a Measure that can contain other Measures.

Reply:
Yep, I've added MMRests to the score tree model and an MMRest measure contains all the measures it replaced within it,